README.md

```diff
@@ -20,45 +20,44 @@ Project
-Basis is a .yaml file that sets these weights. You can find an [easy.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/easy.yaml) in this project folder to get started.
- * Additional instructions are at the start of the file. Open with a text editor.
- * Configuration options in the host.yaml file.
- * Allows a new Mode called "Meta-Mystery", allowing certain mystery settings to apply to all players.
-   * For example, everyone gets the same but random goal.
+Basis is a yaml file that sets these weights. You can find an [easy.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/easy.yaml) in this project folder to get started
+ * Additional instructions are at the start of the file. Open with a text editor
+ * Configuration options can be found in the [host.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/host.yaml) file
+ * Allows a new Mode called "Meta-Mystery", allowing certain mystery settings to apply to all players
+   * For example, everyone gets the same but random goal
  
  MultiServer.py
-  * Supports automatic port-forwarding, can be enabled in host.yaml
-  * improved `!players` command, mentioning how many players are currently connected of how many expected and who's missing
-  * /forfeit Playername now works when the player is not currently connected
-  * Added `/hint` and `!hint`, configuration in host.yaml and description in help
-  * various commands, like /send and /hint use "fuzzy text matching", no longer requiring you to enter a location, player name or item name perfectly
+  * Supports automatic port-forwarding, can be enabled in [host.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/host.yaml)
+  * Added commands `/hint` and `!hint`. See [host.yaml](https://github.com/Berserker66/MultiWorld-Utilities/blob/master/host.yaml) for more information
+  * Updates have been made to the following commands:
+    * `!players` now displays the number of connected players, expected total player count, and which players are missing
+    * `forfiet` now works when a player is no longer connected
+    * `/send`, `/hint`, and various other commands now use "fuzzy text matching". It is no longer required to enter a location, player name or item name perfectly
   * Some item groups also exist, so `/hint Bottles` lists all bottle varieties
 
 Mystery.py
- * Defaults to generating a non-race ROM (Bonta's only makes race ROMs at this time)
+ * Defaults to generating a non-race ROM (Bonta's only makes race ROMs at this time).
 If a race ROM is desired, pass --create-race as argument to it
- * When an error is generated due to a broken .yaml file, it now mentions in the error trace which file, line and character is the culprit
- * Option for progressive items, allowing you to turn them off (see easy.yaml for more info)
- * Rom-Option for extendedmsu (see easy.yaml for more info)
```

Comment:
Per the pull request description. I removed this line as it looks like extendedmsu was removed in this [commit](https://github.com/Berserker66/MultiWorld-Utilities/commit/7f66133d379ad661968f107f76172f152174bbed#diff-28113ab2481a9f388e0c8132ca52acb8)